### DISCUSSION TITLE
Grott Send data to Influx disabled

### DISCUSSION THREAD

**Discussion by Suzukithunder** [johanmeijer/grott#513] [General]
Hi, how come grott is throwing this? In Ini I have declared:

influx = True
influx2 = True
ip = 192.168.0.61
port = 8086
token  = "i_RXW976O9CRea7AzE9dOXGsx5trZo_JLK0ZSBUmAz66zukiWmzqrxA7bH__cxRepiyof5mPVH6_UGwdZ3p7r5A=="
org  = "8deadd5f00c13f87"
bucket = "Testinverter"

**Comment by johanmeijer** [comment]
I think you do have to use the "" for token org en bucket

**Comment by Suzukithunder** [comment]
Hi Johan,

I am using "...." around both token, org and bucket. Is that correct or ?

**Reply by johanmeijer** [reply]
Sorry, you should not use the "..."  I made a typo!

**Reply by Suzukithunder** [reply]
Thanks Johan. It is unfortunately still not connecting

**Reply by johanmeijer** [reply]
Hmm strange. This is how it works with my config: 

    influx = True
    influx2 = True
    ip = 192.168.0.211  
    port = 8086
    token  = FrMTwJiUj9jU2Xy3tGDUCBsIMxb90DGNWlXsAEmj1dW7SwwCmlOJFbkehnxzrPwklXrDlIxXZic0YdhkijPNWg==
    org  = ledidobe
    bucket = grottdb

**Comment by Suzukithunder** [comment]
During initialization it writes the below, even though I have updated the .ini file? Should I pay attention to this?

_Influxdb:
        influx:                  False
        influx2:                 False
        database:                grottdb
        ip:                      localhost
        port:                    8086
        user:                    grott
        password:                **secret**
        organization:            grottorg
        bucket:                  grottdb
        token:                   **secret**
_Extension:
        extension:               False
        extname:                 grottext
        extvar:                  {'none': 'none'}


Grott proxy mode started
         **********************************************************************************
         - Grott - libscrc not installed, no CRC checking only record validation on length!
         **********************************************************************************
Hostname : Thepi
IP :  127.0.1.1 , port :  5279

**Comment by Suzukithunder** [comment]
Hi Johan, it works now. Wrong placement of .ini file. I am trying to incorporate the batterymodule and groboost. I am unable to match the register key numbers to your code (looking at version 1.24). Do you have a hint, so I can start to decipher it ?
